core/src/main/java/org/frankframework/validation/AbstractXmlValidator.java

```diff
@@ -182,7 +182,7 @@ public ValidationResult finalizeValidation(AbstractValidationContext context, Pi
 		if (isThrowException()) {
 			throw new XmlValidatorException(fullReasons, t);
 		}
-		log.warn("{}validation failed: {}", getLogPrefix(session), fullReasons, t);
+		log.warn(getLogPrefix(session) + "validation failed: " + fullReasons, t);
```

Comment:
Fixes `WARN found 2 argument placeholders, but provided 3 for pattern '{}validation failed: {}'`